UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Controllers/BaseController.cs

```diff
@@ -40,6 +44,8 @@ protected IActionResult BuildBadRequestErrorResponse(List<Error> errors)
 
         protected IActionResult BuildInternalServerErrorResponse()
         {
+            Logger.LogError(EventIds.InternalServerError.ToEventId(), "InternalServerError");
```

Comment:
Kindly pass GetCurrentCorrelationId() to LogError method

Reply:
resolved and it will reflect in https://github.com/UKHO/exchange-set-service/pull/29